rancher2/structure_cluster_v2.go

```diff
@@ -26,7 +28,7 @@ type ClusterV2 struct {
 
 func flattenClusterV2(d *schema.ResourceData, in *ClusterV2) error {
 	if in == nil {
-		return nil
+		return fmt.Errorf("[ERROR] flattening cluster: Input cluster is nil")
```

Comment:
We should make an EDI issue for this because `[ERROR]` as an error prefix is entirely non standard.

Reply:
The error message isn't the place to do the formatting, it's too tightly coupling the error message to the output.